ow_system_plugins/base/bol/user_service.php

```diff
@@ -1985,4 +1985,39 @@ public function getQueryFilter( array $tables, array $fields, $params = array()
     {
         return $this->userDao->getQueryFilter($tables, $fields, $params);
     }
+
+    /**
+     * Sends a custom email notification when a profile is re-activated by admin.
+     * @param array $userIds
+     * @return void
+     */
+    public function sendReActivationNotification( array $userIds )
+    {
+        if ( is_array($userIds) )
+        {
+            foreach ( $userIds as $item => $userId )
+            {
+                $user = $this->findUserById($userId);
+                if ( !$user )
+                {
+                    return false;
+                }
+
+                $language = OW::getLanguage();
+                try
+                {
+                    $mail = OW::getMailer()->createMail();

```

Comment:
use BOL_UserService::getDisplayName both for user_reactivation_mail_txt and user_reactivation_mail_html